packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -55,6 +55,38 @@ describe('Validation at REST level', () => {
     }
   }
 
+  @model()
+  class ProductWithCode {
+    @property()
+    id: number;
+
+    @property({
+      required: true,
+      jsonSchema: {
+        format: 'only-alphanumeric',
+        errorMessage: {format: 'Name must not have special characters'},
+      },
```

Comment:
Internally, I understand why it's placed under `jsonSchema`. However, this seems a bit misleading as it's not representative of the JSON schemas defined in OAS 3. Though this may be out-of-scope of this PR.